mapzen-places-api/src/main/java/com/mapzen/places/api/internal/PlaceDialogFragment.java

```diff
@@ -0,0 +1,92 @@
+package com.mapzen.places.api.internal;
+
+import com.mapzen.places.api.R;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.DialogInterface;
+import android.os.Bundle;
+
+/**
+ * Confirmation dialog shown to the user when a POI has been selected.
+ */
+public class PlaceDialogFragment extends DialogFragment implements DialogInterface.OnClickListener {
+  public static final String TAG = PlaceDialogFragment.class.getSimpleName();
+
+  private static final String ARG_NAME_TITLE = "title";
```

Comment:
To be more consistent with the property set on the dialog and with the `setMessage` method, we should change this to `ARG_NAME_MESSAGE`